### PR TITLE
TD-733: Bugfix to prevent dispatching of multiple redux actions

### PR DESCRIPTION
**Pull request by kernelkiller**
- cart related pipeline responses not only dispatch one action (success OR error) when response messages are present

### CODE REVIEW

File: cart/helpers/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { PIPELINE_RESPONSE_MESSAGE_TYPE_ERROR } from '../constants';
+
+/**
+ * Checks if the messages array of a pipeline response contains error messages.
```
Comment (richardgorman) [inline cart/helpers/index.js]: This comment is not what this function does.

File: cart/constants/index.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ export const COUPON_TYPE_PERCENTAGE = 'percentage';
 
 export const COUPON_PUSH_NOTIFICATION = 'cart_add_coupon';
 
+export const PIPELINE_RESPONSE_MESSAGE_TYPE_ERROR = 'error';
```
Comment (richardgorman) [inline cart/constants/index.js]: Could be simplified to `MESSAGE_TYPE_ERROR`,

Comment (kernelkiller) [reply cart/constants/index.js]: done

File: cart/helpers/index.js (hunk 1 above)
Comment (kernelkiller) [reply cart/helpers/index.js]: As discussed i'll leave it like this

**Pull request merged**